data_steward/cdr_cleaner/cleaning_rules/store_new_duplicate_measurement_concept_ids.py

```diff
@@ -102,7 +102,7 @@
       COALESCE(CASE
           WHEN dupes.concept_vocab_in_table NOT IN ('LOINC', 'NAACCR') THEN
            ( SELECT MAX(value_as_concept_id) FROM and_cn_dupes AS t2 WHERE 
-           t2.concept_name_in_table = dupes.concept_name_in_table AND t2.concept_vocab_in_table = 'LOINC' )
+           lower(t2.concept_name_in_table) = lower(dupes.concept_name_in_table) AND t2.concept_vocab_in_table = 'LOINC' )
```

Comment:
@ksdkalluri I had to lower these as well to have the test pass.

Without this change 100003 was inserted into the table but no change to 'aou_standard_vac'. 
Result was (100003, 'negative', 'SNOMED', 100003) whereas it should be (100003, 'negative', 'SNOMED', 100001)

 Does this change make sense to you?